code/dynamic_programming/src/longest_common_substring/Longest_Common_Substring.cpp

```diff
@@ -0,0 +1,72 @@
+// Space Complexity: O(n)
+// Time Complexity: O(m*n) 
+
+#include <string>
+#include <iostream>
+#include <vector> 
+using namespace std;
```

Comment:
Don't use this.

Reply:
@arnavb I didn't get it. Are you saying about the header files not to use or for `using namespace std`?

Reply:
I'm talking about `using namespace`. I know many files have that, but it is before I started maintaining for that line. It is bad practice, so please remove it.

Reply:
I have removed it. Please have a look at it.